tools/wptrunner/wptrunner/wpttest.py

```diff
@@ -158,13 +158,15 @@ class Test(object):
     long_timeout = 60  # seconds
 
     def __init__(self, tests_root, url, inherit_metadata, test_metadata,
-                 timeout=None, path=None, protocol="http", quic=False):
+                 timeout=None, path=None, protocol="http", subdomain=None,
+                 quic=False):
         self.tests_root = tests_root
         self.url = url
         self._inherit_metadata = inherit_metadata
         self._test_metadata = test_metadata
         self.timeout = timeout if timeout is not None else self.default_timeout
         self.path = path
+        self.subdomain = subdomain
```

Comment:
I would be interested in @jgraham 's thoughts on if this should be in `environment`. (I'm not really sure why we have the `environment` dict. I think this being a member makes sense, but maybe there's some reason behind `environment` I'm not aware of).

Reply:
Yeah, good comment. I had originally planned on piping this through the `environment` dict (because I'm not super familiar with anything really... and it seemed like an OK place to stick stuff), but James commented on that plan here: https://github.com/web-platform-tests/wpt/issues/20932#issuecomment-714472782